practices/managing-static-assets.md

```diff
@@ -0,0 +1,20 @@
+# Managing static assets
+
+By **static assets** we mean webfonts, JavaScript libraries (like jQuery or Mathjax), and any other files that get loaded by the browser from a domain that is not ours. These static assets are characterised by:
+
+* Being essential or important for the working of a page.
+* Being updated infrequently.
+* Require some manual intervention in order to be updated.
+
+Usually these assets are added to the page as part of the application, and not through tag managers.
+
+For this kind of static resources is our choice to **self-host them in our servers** instead of including them on our pages from an external CDN.
+
+## Benefits of self-hosting static assets
+
+* **Resilience.** As long as our site is up, the resource will be up. We don't depend on other servers or additional CDNs and we won't be affected by their outages.
+* **Consistency.** Static assets are cached and served in exactly the same way that the reset of our resources are, and they use the same CDN. As a result, timings and user experience is more consistent all around the world.
+* **Risk** that the service hosting the asset shuts down in the future, forcing us to upgrade a bunch of apps, some of which may be unmaintained.
```

Comment:
```suggestion
* **Security.** reduces our front-end [attack surface](https://en.wikipedia.org/wiki/Attack_surface).
```

Reply:
This "Risk" point probably covered by slightly expanding the first point in this list, as per following suggestion.